app/src/main/java/com/kms/cura/view/fragment/RegDoc_PersonalFragment.java

```diff
@@ -0,0 +1,187 @@
+package com.kms.cura.view.fragment;
+
+import android.app.AlertDialog;
+import android.app.DatePickerDialog;
+import android.app.Dialog;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.DatePicker;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.Spinner;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.kms.cura.R;
+import com.kms.cura.utils.InputUtils;
+import com.kms.cura.view.activity.RegisterDoctorActivity;
+import com.kms.cura.view.adapter.StringListAdapter2;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+
+public class RegDoc_PersonalFragment extends Fragment implements View.OnClickListener {
+
+    private EditText edtFName, edtLName;
+    private String FName, LName;
+    private int sex;
+    private int day, month, year;
+    private Calendar calendar;
+    private TextView tvBirth;
+    private ImageButton btnChooseDate, btnNext;
+    private Spinner spnSex;
+
+    public RegDoc_PersonalFragment() {
+        setCurrentDate();
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        View myFragmentView = inflater.inflate(R.layout.fragment_reg_doc__personal, container, false);
```

Comment:
onCreateView is too crowded. Split it into sub methods like initView, initButtons, initData...